src/keycloak/index.js

```diff
@@ -9,6 +11,14 @@ let initOptions = {
     pkceMethod: 'S256'
 };
 
+// For some reason idpHint cannot be specified in the Keycloak constructor or init options.
+// https://stackoverflow.com/a/56338011/201891
+let kcLogin = keycloak.login;
+keycloak.login = (options) => {
+    options.idpHint = 'idir';
+    return kcLogin(options);
+};
+

```

Comment:
I briefly tried to load this from the configuration file, and it actually worked, but at this time I'm uncomfortable putting async code in a module export. e.g. We export `keycloak`, but when is it actually ready to use? It worked when I tested it, but was that just lucky timing?

Config is something we should probably work on.